Source/SteveSloth/Private/MyPlayer.cpp

```diff
@@ -10,41 +10,64 @@
 
 #include "MyPlayer.h"
 
-// Sets default values
 AMyPlayer::AMyPlayer()
 {
- 	// Set this character to call Tick() every frame.  You can turn this off to improve performance if you don't need it.
 	PrimaryActorTick.bCanEverTick = true;
-
-	//Setting Default Values:
+	
 	MaxHealth = 0;
 	CurrentHealth = 0;
 	GrubsCollected = 0;
 	CurrentSlingshotAmmo = 0;
-	LevelsCompleted = 0;
-	BossesKilled = 0;
 	LeavesFound = 0;
-
 }
 
-// Called when the game starts or when spawned
 void AMyPlayer::BeginPlay()
 {
 	Super::BeginPlay();
 	
 }
 
-// Called every frame
 void AMyPlayer::Tick(float DeltaTime)
 {
 	Super::Tick(DeltaTime);
 
 }
 
-// Called to bind functionality to input
 void AMyPlayer::SetupPlayerInputComponent(UInputComponent* PlayerInputComponent)
 {
 	Super::SetupPlayerInputComponent(PlayerInputComponent);
 
+	if (APlayerController* PlayerController = Cast<APlayerController>(GetController()))
+	{
+		if (UEnhancedInputLocalPlayerSubsystem* Subsystem = ULocalPlayer::GetSubsystem<UEnhancedInputLocalPlayerSubsystem>(PlayerController->GetLocalPlayer()))
+		{
+			Subsystem->ClearAllMappings();
+			Subsystem->AddMappingContext(InputMapping, 0);
+		}
+	}
+
+	if (UEnhancedInputComponent* inputComponent = Cast<UEnhancedInputComponent>(PlayerInputComponent))
+	{
+		inputComponent->BindAction(ForwardBack, ETriggerEvent::Triggered, this, &AMyPlayer::MoveForwardBack);
+		inputComponent->BindAction(ForwardBack, ETriggerEvent::Completed, this, &AMyPlayer::MoveForwardBack);
+
+		inputComponent->BindAction(Turn, ETriggerEvent::Triggered, this, &AMyPlayer::TurnLeftRight);
+		inputComponent->BindAction(Turn, ETriggerEvent::Completed, this, &AMyPlayer::TurnLeftRight);
+	}
 }
 
+void AMyPlayer::MoveForwardBack(const FInputActionValue& Value)
+{
+	float const Direction = Value.Get<float>();
+	FVector const Forward = GetActorForwardVector();
+	AddMovementInput(Forward, Direction);
+}
+
+void AMyPlayer::TurnLeftRight(const FInputActionValue& Value)
```

Comment:
Rename thins to MoveLeftRight and add a comment saying move and turn in that direction